data/afaanoromoo/bbc_afaanoromo_radio/liveradio.json

```
@@ -1,29 +1,64 @@
 {
-    "metadata": {
-        "id": "urn:bbc:ares::asset:afaanoromo/bbc_afaanoromo_radio/liveradio",
-        "tags": {}
+  "metadata": {
+    "id": "urn:bbc:ares::ws_live:bbc_afaanoromoo_radio",
+    "locators": {},
+    "type": "WS-LIVE",
+    "createdBy": "afaanoromoo",
+    "language": "id-ID",
+    "lastUpdated": 0,
+    "firstPublished": 0,
+    "lastPublished": 0,
+    "options": {},
+    "analyticsLabels": {
+      "pageIdentifier": "afaanoromoo.bbc_afaanoromoo_radio.liveradio.page",
+      "pageTitle": "Dhaggeeffadhaa - BBC News Afaanoromoo"
     },
-    "content": {
-        "blocks": [
-            {
-                "id": "liveradio",
-                "type": "media",
-                "format": "audio",
-                "title": "TODO",
-                "caption": "TODO",
-                "versions": [
-                    {
-                        "versionId": "liveradio"
-                    }
-                ]
-            }
-        ]
-    },
-    "promo": {
-        "subType": "IDX",
-        "name": "TODO: Afaanoromo Live Radio",
-        "uri": "/afaanoromo/bbc_afaanoromo_radio/liveradio",
-        "id": "TODO",
-        "type": "simple"
+    "tags": {},
+    "version": "1.1.1",
+    "blockTypes": ["heading", "version", "paragraph"]
+  },
+  "content": {
+    "blocks": [
+      {
+        "text": "Dhaggeeffadhaa",
+        "markupType": "plain_text",
+        "type": "heading"
+      },
+      { "text": "Sagantaawwan keenya.", "type": "paragraph" },
+      {
+        "id": "liveradio",
+        "subType": "primary",
+        "format": "audio",
+        "externalId": "bbc_afaanoromoo_radio",
```

```suggestion
        "externalId": "bbc_afaanoromoo_radio",
```

If we are having the mapping for `bbc_afaanoromoo_radio` to `bbc_oromo_radio` in simorgh this is likely where we would want to change it